autoinstall/Pardus/preseed.cfg

```diff
@@ -0,0 +1,250 @@
+# warning. This can be preseeded away...
+d-i partman-lvm/device_remove_lvm boolean true
+# The same applies to pre-existing software RAID array:
+d-i partman-md/device_remove_md boolean true
+# And the same goes for the confirmation to write the lvm partitions.
+d-i partman-lvm/confirm boolean true
+d-i partman-lvm/confirm_nooverwrite boolean true
+
+# You can choose one of the three predefined partitioning recipes:
+# - atomic: all files in one partition
+# - home:   separate /home partition
+# - multi:  separate /home, /var, and /tmp partitions
+d-i partman-auto/choose_recipe select atomic
+
+# This makes partman automatically partition without confirmation, provided
+# that you told it what to do using one of the methods above.
+d-i partman-partitioning/confirm_write_new_label boolean true
+d-i partman/choose_partition select finish
+d-i partman/confirm boolean true
+d-i partman/confirm_nooverwrite boolean true
+
+# Force UEFI booting ('BIOS compatibility' will be lost). Default: false.
+{% if firmware is defined and firmware == 'efi' %}
+d-i partman-efi/non_efi_system boolean true
+
+# Ensure the partition table is GPT - this is required for EFI
+d-i partman-partitioning/choose_label string gpt
+d-i partman-partitioning/default_label string gpt
+{% endif %}
+
+# This makes partman automatically partition without confirmation.
+d-i partman-md/confirm boolean true
+d-i partman-partitioning/confirm_write_new_label boolean true
+d-i partman/choose_partition select finish
+d-i partman/confirm boolean true
+d-i partman/confirm_nooverwrite boolean true
+
+# Add offical repo
+d-i apt-setup/local0/repository string deb http://depo.pardus.org.tr/pardus yirmibir main contrib non-free
+d-i apt-setup/local0/source boolean false
+d-i apt-setup/local1/repository string deb http://depo.pardus.org.tr/guvenlik yirmibir main contrib non-free
+d-i apt-setup/local1/source boolean false
+
+# Default repository information (don't include codename data, d-i figures it
+# out from what's available in the ISO)
+#d-i mirror/country string manually
+#d-i mirror/http/hostname string depo.pardus.org.tr
+#d-i mirror/http/hostname seen false
+#d-i mirror/http/directory string /pardus
+#d-i mirror/http/proxy string
+# Don't add any security and updates repo to avoid an upgrade during installation
+# We might need these later, think about the following lines.
+#d-i apt-setup/services-select multiselect
+#d-i apt-setup/security_host string depo.pardus.org.tr
+#d-i apt-setup/security_path string /guvenlik
+
+### Apt setup
+d-i apt-setup/use_mirror boolean false
+d-i apt-setup/cdrom/set-first boolean false
+d-i apt-setup/cdrom/set-next boolean false
+d-i apt-setup/cdrom/set-failed boolean false
+d-i apt-setup/disable-cdrom-entries boolean true
+d-i apt-setup/enable-source-repositories boolean true
+# You can choose to install non-free and contrib software.
+d-i apt-setup/non-free boolean true
+d-i apt-setup/contrib boolean true
```

Comment:
Is there any package need to be installed from non-free or contrib repo? If not, then no need to enable them.

Reply:
remove it. Thanks